### PR TITLE
Itm 619 compare sim vs text

### PR DESCRIPTION
**Pull request by kaitlyn-sharo**
Get alignment comparisons for the simulator vs text scenarios

run the probe matcher. You'll notice this runs separately from the rest of the code. ADEPT may not run properly right now due to missing session ids in the server.

### CODE REVIEW

File: deployment_script.py [hunk 1]
```diff
@@ -10,12 +10,13 @@
...
-        return True 
-    # return true if it is a newer db version
-    return db_version > version_obj['version']
+    return True
+    # collection = mongoDB[VERSION_COLLECTION]
+    # version_obj = collection.find_one()
+    # if version_obj is None:
+    #     return True 
+    # # return true if it is a newer db version
+    # return db_version > version_obj['version']
```
Comment (nextcen-dgemoets) [inline deployment_script.py:19]: I assume we don't want to merge this change?

File: dre_probe_matcher.py [hunk 2]
```diff
@@ -6,8 +6,9 @@
 from decouple import config 
 
 SEND_TO_MONGO = True # send all raw and calculated data to the mongo db if true
-RUN_ALIGNMENT = True # send data to servers to calculate alignment if true
+RUN_ALIGNMENT = False # send data to servers to calculate alignment if true
```
Comment (nextcen-dgemoets) [inline dre_probe_matcher.py:9]: Is this a deliberate change to the default?  Or just something you wanted to do locally?

**Review by nextcen-dgemoets — COMMENTED** [review]
> ADEPT may not run properly right now due to missing session ids in the server

Is this what caused dozens of errors like:
- `time data 'Timestamp' does not match format '%m/%d/%Y %I:%M:%S %p'`

File: deployment_script.py (hunk 1 above)
Comment (kaitlyn-sharo) [reply deployment_script.py:19]: ugh...how many times do I have to make that mistake before I remember!

**Comment by kaitlyn-sharo** [comment]
> > ADEPT may not run properly right now due to missing session ids in the server
> 
> Is this what caused 101 errors like:
> 
> * `time data 'Timestamp' does not match format '%m/%d/%Y %I:%M:%S %p'`

I don't think so...I never got that, I'll look into it

**Review by nextcen-dgemoets — APPROVED** [review]
The data was populated in the DB and in the table.

**Review by phile-caci — APPROVED** [review]
Ran this after: https://github.com/NextCenturyCorporation/itm-ingest/pull/58

No issues with Adept scenarios

Saw expected? warnings about Probe 9 and 9214/9112/9110 scenarios
Looks good!

**Review by brianpippin — APPROVED** [review]
I ran 629, 628, then probe matcher with no issues.

**Pull request merged**